packages/shared/src/buttons/Button.js

```diff
@@ -44,14 +50,13 @@ const styleSheet = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center',
     paddingHorizontal: 11,
+    height: 44,
+    borderRadius: 6,
+  },
+  primary: {
     backgroundColor: defaultTokens.paletteProductNormal,
-    android: {
```

Comment:
Are we sure that there are no differences between Android and iOS anymore?

Reply:
Only text-transform seems to be different, but that's another component.